packages/gutenpride-template/src/src/edit.js.mustache

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress components that create the necessary UI elements for the block
+ *
+ * @see https://github.com/WordPress/gutenberg/tree/master/packages/components
```

Comment:
We can use the developer portal for consistancy.
```suggestion
 * @see https://developer.wordpress.org/block-editor/packages/packages-components/
```

Reply:
This is done :)